src/services/hondaApi.tsx

```diff
@@ -0,0 +1,22 @@
+import { createApi, fetchBaseQuery } from '@reduxjs/toolkit/query/react';
+
+export const hondaApi = createApi({
+  reducerPath: 'hondaApi',
+  baseQuery: fetchBaseQuery({
+    baseUrl: 'https://18qv4o7xm9.execute-api.eu-west-1.amazonaws.com',
```

Comment:
please extract information that can vary into environment variables

Reply:
these env variables should live in `.env` files and should not be under version controls, cause they potentially contain secrets like logins, passwords etc.